### PR TITLE
master-lps-131955

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-131919
https://issues.liferay.com/browse/LPS-131955

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/liferay-lima/liferay-portal/pull/1811<br></br>Please see changes [here.](https://github.com/austinchiang/liferay-portal/compare/3f54e7af8ff9~2...3f54e7af8ff9)

**Comment by austinchiang** [comment]
Nice work @liuxingye! Since adding/accessing a category is a really basic test we should make sure to test the UI for it somewhere, so I kept the acceptance test to do that. We can apply openURL to all other tests using categories navigation

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/questions/QuestionsWidgetActions.testcase [hunk 1]
```diff
@@ -371,28 +226,131 @@ definition {
...
 	@priority = "4"
-	@refactorneeded
-	test CancelDeletions {
+	@refactordone
+	test CanAddQuestionWithDepotImage {
+		JSONDepot.addDepot(
+			depotDescription = "This is the description of a depot",
+			depotName = "Test Depot Name");
+
+		DepotNavigator.openDepotAdmin();
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/questions/QuestionsWidgetActions.testcase:237]: @liuxingye I forgot to remove this line before sending the pull to BChan. Could you remove it in another ticket? You can use the same LPS #

Comment (liuxingye) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/questions/QuestionsWidgetActions.testcase:237]: ok. I will remove it

**Pull request closed without merging**